app/models/evaluation.rb

```diff
@@ -67,4 +67,52 @@ def self.filters_to_json
       }
     ].to_json
   end
+
+  def self.to_csv(ids = [])
+
+    csv = ''
+
+    evaluation_columns = Evaluation.new.attributes.keys
+    evaluation_columns << "evaluation_id"
+
+    excluded_attributes = ["created_at", "updated_at", "id", "site_id", "source_id"]
+
+    evaluation_columns.delete_if { |k, v| excluded_attributes.include? k }
+
+    evaluation_columns << "wpda_id"
+    evaluation_columns << "iso3"
+    evaluation_columns << "name"
+    evaluation_columns << "designation"
+    evaluation_columns << "source_data_title"
+    evaluation_columns << "source_resp_party"
+    evaluation_columns << "source_year"
+    evaluation_columns << "source_language"
+
+    csv << evaluation_columns.join(',')
+    csv << "\r\n"
```

Comment:
Instead of generating the csv like so, which requires manual addition of those extra characters, you could try to use the CSV ruby library.
Inside the `CSV.open(filename, 'w') do |csv|` block you only need to do `csv << line` without paying attention to newlines as it will be done automatically for you.
Worth a try

Reply:
Nice! Great idea!